tests/dom_tests.js

```diff
@@ -174,6 +174,11 @@ QUnit.test("matches", function(assert) {
   assert.ok(document.querySelector('#baz').matches('.beta'));
 });
 
+QUnit.test("closest", function(assert) {
+  assert.equal(document.querySelector('#bar').closest('#foo'), document.querySelector('#foo'));
+  assert.equal(document.querySelector('#baz').closest('#foo'), document.querySelector('#foo'));
+});
```

Comment:
I don't use node myself. You can just hit load the test file directly in a browser, or e.g.:

https://rawgit.com/inexorabletash/polyfill/master/tests/dom.html

And yes, getting continuous integration would be swell. As noted I don't use node myself for any projects, but happy to field further PRs with tests, if there are appropriate instructions.